tools/launch_ccm_cluster.py

```diff
@@ -258,60 +261,43 @@ def _start(self, config):
         if not response:
-            raise Exception('CCM cluster creation request failed')
+            raise ClusterActionException('CCM cluster creation request failed')
         response_content = response.read().decode('utf-8')
         response_json = json.loads(response_content)
         logger.info('Launch response:\n{}'.format(pprint.pformat(response_json)))
         cluster_id = int(response_json.get('id', 0))
         if not cluster_id:
-            raise Exception('No Cluster ID returned in cluster creation response: {}'.format(response_content))
+            raise ClusterActionException('No Cluster ID returned in cluster creation response: {}'.format(response_content))
         stack_id = response_json.get('stack_id', '')
         if not stack_id:
-            raise Exception('No Stack ID returned in cluster creation response: {}'.format(response_content))
+            raise ClusterActionException('No Stack ID returned in cluster creation response: {}'.format(response_content))
 
         cluster_info = self.wait_for_status(
             cluster_id,
             ['CREATING', 'RUNNING_NEEDS_INFO'], # pending states
             'RUNNING', # desired state
             config.start_timeout_mins)
         if not cluster_info:
-            raise Exception('CCM cluster creation failed or timed out')
+            raise ClusterActionException('CCM cluster creation failed or timed out')
         dns_address = cluster_info.get('DnsAddress', '')
         if not dns_address:
-            raise Exception('CCM cluster_info is missing DnsAddress: {}'.format(cluster_info))
+            raise ClusterActionException('CCM cluster_info is missing DnsAddress: {}'.format(cluster_info))
         logger.info('Cluster is now RUNNING: {}'.format(cluster_info))
 
-        if config.mount_volumes:
-            logger.info('Enabling mount volumes for cluster {} (stack id {})'.format(cluster_id, stack_id))
-            # fabric spams to stdout, which causes problems with launch_ccm_cluster.
-            # force total redirect to stderr:
-            stdout = sys.stdout
-            sys.stdout = sys.stderr
-            import enable_mount_volumes
-            enable_mount_volumes.main(stack_id)
-            sys.stdout = stdout
-
         # we fetch the token once up-front because on Open clusters it must be reused.
         # given that, we may as well use the same flow across both Open and EE.
         logger.info('Fetching auth token')
         dcos_url = 'https://' + dns_address
         auth_token = dcos_login.DCOSLogin(dcos_url).get_acs_token()
 
-        if config.permissions:
-            logger.info('Setting up permissions for cluster {} (stack id {})'.format(cluster_id, stack_id))
+        is_enterprise = config.cf_template.startswith('ee.')
+        clustinit = configure_test_cluster.ClusterInitializer(cluster_id,
+                stack_id, auth_token, dns_address, is_enterprise,
+                os.environ.get('SECURITY'))
```

Comment:
It's sort of there already; as self.permissions.
Options to move this there:
1 - make redundant item
2 - change self.permissions to self.security and adjust all current users.